lib/generate.sh

```diff
@@ -107,23 +133,15 @@ function _generate_sysext() {
   local fname="$5"
 
   announce "Creating extension image '${fname}' and generating SHA256SUM"
-  case "$format" in
-    btrfs)
-      mkfs.btrfs --mixed -m single -d single --shrink --rootdir "${basedir}" "${fname}"
-      ;;
-    ext2|ext4)
-      truncate -s "${ext_fs_size}" "${fname}"
-      mkfs."${format}" -E root_owner=0:0 -d "${basedir}" "${fname}"
-      resize2fs -M "${fname}"
-      ;;
-    squashfs)
-      mksquashfs "${basedir}" "${fname}" -all-root -noappend -xattrs-exclude '^btrfs.'
-      ;;
-    erofs)
-      mkfs.erofs "${fname}" "${basedir}"
-      ;;
+  (
```

Comment:
I think we should make signing opt-in, so wrap the the direct invocations in one if branch?

Reply:
This could also reuse the exported repart flag options env var.
FYI: I also noticed problems with erofs and have filed a PR to fix this and if merged earlier you might need to adapt the settings when rebasing here https://github.com/flatcar/sysext-bakery/pull/176